### PR TITLE
Feature/upgrage java17

### PR DESCRIPTION
**Pull request by jianmingtu**
# Description

This PR includes the following proposed change(s):

- {List all the changes, if possible add the jira ticket #}

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/bcsc-auth/pom.xml [hunk 1]
```diff
@@ -157,7 +169,7 @@
 			<plugin>
 				<groupId>org.jacoco</groupId>
 				<artifactId>jacoco-maven-plugin</artifactId>
-				<version>0.8.4</version>
```
Comment (jianmingtu) [inline src/bcsc-auth/pom.xml:160]: upgrade jacoco-maven-plugin to 0.8.8, 0,8.4 causes infinite loop in unittest

File: src/bcsc-auth/pom.xml [hunk 2]
```diff
@@ -86,8 +86,20 @@
 		</dependency>
 		<dependency>
 			<groupId>io.jsonwebtoken</groupId>
-			<artifactId>jjwt</artifactId>
```
Comment (jianmingtu) [inline src/bcsc-auth/pom.xml:89]: Upgrade JWT Authentication to its Java 17 version (Jjwt to  jjwt-impl)

Comment (TayGov) [reply src/bcsc-auth/pom.xml:89]: You can make the same change here as ecrc

**Review by TayGov — CHANGES_REQUESTED** [review]
The one small change same as the one in ecrc api

**Review by TayGov — APPROVED** [review]

**Pull request merged**